observability/logging/logging-6.0/log6x-output.adoc

```diff
@@ -0,0 +1,30 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="log6x-output-types"]
+= Log output types
+:context: logging-output-types
+
+toc::[]
+
+Outputs define the destination where logs are sent to from a log forwarder. You can configure multiple types of outputs in the `ClusterLogForwarder` custom resource (CR) to send logs to servers that support different protocols.
+
+include::modules/log6x-supported-log-outputs.adoc[leveloffset=+1]
+
+[id="log6x-output-types-descriptions"]
+== Output type descriptions
+
+`loki`:: Loki, a horizontally scalable, highly available, multi-tenant log aggregation system.
+`kafka`:: A Kafka broker. The `kafka` output can use a TCP or TLS connection.
+`elasticsearch`:: An external Elasticsearch instance. The `elasticsearch` output can use a TLS connection.
+`fluentdForward`:: An external log aggregation solution that supports Fluentd. This option uses the Fluentd `forward` protocols. The `fluentForward` output can use a TCP or TLS connection and supports shared-key authentication by providing a `shared_key` field in a secret. Shared-key authentication can be used with or without TLS.
++
+[IMPORTANT]
```

Comment:
Does not exist in 6.0

Reply:
Would you like to make these updates as part of our larger PR? Please let me know if you have other thoughts. Thanks!